app/controllers/api/v1/packages_controller.rb

```diff
@@ -181,6 +183,15 @@ def print_barcode
         print_inventory_label
       end
 
+      def package_valuation
```

Comment:
Could you add the apipie controller descriptors above the method ?

e.g
``` ruby
      api :DELETE, "/v1/packages/1", "Delete an package"
      description "Deletion of the Package item in review mode"
      def destroy
          ...
      end
```

We kinda stopped doing that, and now we sometimes can't tell endpoint methods from helper methods 😆 
I'm trying to bring back that habit :)